src/yunohost/app.py

```diff
@@ -236,13 +236,13 @@ def _sanitized_absolute_url(perm_url):
 
             return perm_domain, perm_path
 
-        this_app_perms = {p: i for p, i in permissions.items() if p.startswith(app_id + ".") and i["url"]}
+        this_app_perms = {p: i for p, i in permissions.items() if p.startswith(app_id + ".")}
```

Comment:
Are you sure it's a bug and not a feature ? :/

iirc this was done this way to cover non-webapps like for example the mumbleserver app, for which it doesn't make sense to display a tile because there's no web page to display 

(and also : iirc the `app_map` is what is used to detect url conflicts, so with this new code, for example installing mumbleserver on a domain mean you can't install any other app on that domain)

Reply:
Related to: https://github.com/YunoHost/yunohost/pull/877 and more precisely, app like https://github.com/YunoHost-Apps/lutim_ynh/pull/44.

The situation is as follows:

We have an app that protects url via a regex:
https://github.com/YunoHost-Apps/lutim_ynh/blob/b117e3d5547ef2f5265e59d95f8d88d23b0c998a/scripts/install#L205
(same for [opensondage](https://github.com/YunoHost-Apps/opensondage_ynh/))
To make the application private, we only want to protect the urls in the regex, not the whole application.

We can indeed create a new permission, without url called "public images" or whatever, and do the same job with it :-)